hvantk/enrichex/burden.py

```
@@ -578,22 +961,798 @@ def run_burden_analysis(
+                [gene_lengths.get(g, 1000.0) / 1000.0 for g in sorted_genes],
+                dtype=np.float64,
+            )
+        else:
+            logger.info("Normalizing qualifying matrix by variant site count proxy")
+            _cohort_rows = cohort_mt.rows()
+            gene_site_map = (
+                _cohort_rows.group_by(gene=_cohort_rows[gene_field])
+                .aggregate(_n_sites=hl.agg.count())
+                .to_pandas()
+            )
+            site_dict = dict(zip(gene_site_map["gene"], gene_site_map["_n_sites"]))
```

Ruff (0.15.5)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)